images/os/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fedora:29 AS build
+
+COPY --from=registry.svc.ci.openshift.org/openshift/origin-v4.0:machine-os-content /srv/ /srv/
+RUN yum install -y ostree yum-utils selinux-policy-targeted && \
+    commit=$( find /srv -name *.commit | sed -Ee 's|.*objects/(.+)/(.+)\.commit|\1\2|' | head -1 ) && \
+    mkdir /tmp/working && cd /tmp/working && \
+    yumdownloader --destdir=/tmp/rpms origin-node origin-clients && \
+    for i in $(find /tmp/rpms/ -name origin-*); do rpm2cpio $i | cpio -div; done && \
+    ostree --repo=/srv/repo commit --parent=$commit --tree=dir=. --orphan --selinux-policy / \
+        -s "origin-ci-dev overlay RPMs" --branch=origin-ci-dev
```

Comment:
This needs to include the previous tree too so that it's a complete OSTree. See similar code in rpm-ostree: https://github.com/projectatomic/rpm-ostree/blob/master/tests/vmcheck/overlay.sh. It's a mouthful, but essentially:

```
ostree checkout <parent-tree> /tmp/working
<overlay binaries>
ostree commit /tmp/working --branch=origin-ci-dev
```

Reply:
Oh.

Reply:
Actually, just `ostree commit --tree=ref=<parent-tree> --tree=dir=.` should work too.

Reply:
Ok, add --tree-ref

Reply:
--tree-ref=$commit?

Reply:
`--tree=ref=$commit --tree=dir=.` will basically start off with the same tree as `$commit` and then overlay `.` on top of that.